core/src/main/java/hudson/Util.java

```diff
@@ -1579,7 +1558,10 @@ public static void closeAndLogFailures(@CheckForNull Closeable toClose, @Nonnull
         try {
             toClose.close();
         } catch(IOException ex) {
-            logger.log(Level.WARNING, String.format("Failed to close %s of %s", closeableName, closeableOwner), ex);
+            LogRecord record = new LogRecord(Level.WARNING, "Failed to close {0} of {1}");
```

Comment:
4x the code for the frequent case when WARNING doesn't get logged? Really?

Reply:
Should I revert this change?